ops/testnet/prod/core/main.tf

```diff
@@ -462,7 +462,36 @@ module "relayer" {
   ingress_ipv6_cdir_blocks = []
   service_security_groups  = flatten([module.network.allow_all_sg, module.network.ecs_task_sg])
   cert_arn                 = var.certificate_arn_testnet
-  container_env_vars       = local.relayer_env_vars
+  container_env_vars       = merge(local.relayer_env_vars, { RELAYER_SERVICE = "polling" })
```

Comment:
This module will not have http server and, will not be able to respond to ping.

Reply:
could you update that?